### DISCUSSION TITLE
Parcel creates two .js files

### DISCUSSION THREAD

**Discussion by jbonigomes** [parcel-bundler/parcel#5018] [Q&A]
When running 'parcel index.html', where my 'index.html' imports a single .js file, Parcel's output to the dist folder contains two .js files, one imported in the header section and the other at the end of the body. Is there a way to get Parcel to generate a single .js file? Same goes for .css...

**Comment by GoAndrew** [comment]
@jbonigomes `parcel build index.html`

**Reply by jbonigomes** [reply]
Thanks for reaching out, unfortunately the build command yields the same result (two .js and two .css files).

**Comment by spankyed** [comment]
Same issue here.

**Reply by blueromans** [reply]
Same issue

**Reply by alexsoin** [reply]
the problem is relevant. The first file is compiled in its original form, the second with assembly changes

**Comment by kirstenhh** [comment]
I have this as well, and it's a problem for me. Is there a way to either prevent the second one from getting built, or to name the two differently so we can distinguish them at a glance? (I'm using parcel-namer-custom, but there's no way to handle the two outputs as far as I can tell.)

**Reply by FDiskas** [reply]
add a parameter to build script `--no-scope-hoist`

**Comment by mischnic** [comment]
Parcel automatically creates two scripts for differential serving via module/nomodule: https://parceljs.org/blog/rc0/#differential-bundling

If you set a `browserslist` config that is new enough that you don't support any old browsers (so you specify browsers with module support), then there will also be only one JS bundle.

**Reply by AliMammadli** [reply]
I tried `"browserslist": "fully supports es6-module"` but not worked. Is this config incorrect or parcel does not support?
